fathom/src/surface/elaboration.rs

```diff
@@ -2177,6 +2445,7 @@ impl<'interner, 'arena, 'error> Context<'interner, 'arena, 'error> {
                         );
 
                         core::Term::Let(
+                            (&range).into(), // FIXME: is this the right range to use here?
```

Comment:
This is tricky, as it's sort of ‘generated code’… so it’s a little hard to give a precise range… it should probably at least be the range of the match arm… currently this is just the pattern 🤔